lightning/src/blinded_path/message.rs

```diff
@@ -138,6 +138,12 @@ impl BlindedMessagePath {
 		&self.0.blinded_hops
 	}
 
+	#[cfg(test)]
+	/// The [`BlindedPath`] within the [`BlindedMessagePath`]
+	pub(crate) fn inner_blinded_path(&self) -> &BlindedPath {
+		&self.0
+	}
+
```

Comment:
Doesn't look like this is needed as we can use `blinded_hops` instead for both message and payment paths.